package.json

```diff
@@ -75,6 +76,7 @@
     "prettier-loader": "^3.3.0",
     "sass-loader": "^13.0.0",
     "sort-css-media-queries": "^2.0.0",
+    "source-map-explorer": "^2.5.2",
```

Comment:
I thought this should be a dev dependency?

Reply:
But it is a dev dependency :)
https://github.com/neos/Neos.NeosIo/blob/088007fa03730ac4ed3ce6527bba6a3c15de7701/package.json